defaults.yml

```diff
@@ -158,9 +158,9 @@ apt_repos:
   erlang:
     repo: http://packages.erlang-solutions.com/debian
     key_url: https://packages.erlang-solutions.com/debian/erlang_solutions.asc
-#  aptly:
-#    repo: https://apt-mirror.openstack.blueboxgrid.com/aptly/squeeze
-#    key_url: https://apt-mirror.openstack.blueboxgrid.com/keys/aptly.key
+  # aptly:
```

Comment:
actually we could just delete these lines just in there from legacy copy/paste BS.